src/components/ProductPage/ProductWalletBalance.tsx

```diff
@@ -40,4 +58,9 @@ const StyledTokenBalance = styled.div`
   color: ${({ theme }) => theme.colors.grey[500]};
 `
 
+const AddToMetamask = styled.div`
+  display: inline-block;
+  float: right;
```

Comment:
In general I'd advise against using float. It's kind of hard to be precise with it. Flexbox should solve all our needs